lms/envs/aws.py

```diff
@@ -437,6 +437,9 @@
 # Course Forums Download
 COURSE_FORUMS_DOWNLOAD_ROUTING_KEY = HIGH_MEM_QUEUE
 
+#Student Forums Download
+STUDENT_FORUMS_DOWNLOAD_ROUTING_KEY = HIGH_MEM_QUEUE
```

Comment:
This should actually try to load the value from `ENV`, falling back to the value from `lms/envs/common.py`.